https://issues.redhat.com/browse/ACM-16519 Clarify Placement (#7395)
open-cluster-management/rhacm-docs: clusters/cluster_lifecycle/placement_decision.adoc

[#placement-decision]
= Checking selected `ManagedClusters` by using `PlacementDecisions`

One or more `PlacementDecision` kinds with the label `cluster.open-cluster-management.io/placement={placement_name}` are created to represent `ManagedClusters` selected by a placement.

> If a `ManagedCluster` is selected and added to a `PlacementDecision`, components that consume this placement might apply the workload on this `ManagedCluster`. 

Reviewer: What do you think of more user focus here?: 

```suggestion
If you select `ManagedCluster` and add it to a `PlacementDecision`, the components that consume this placement might apply the workload on this `ManagedCluster`. 
```

> After the `ManagedCluster` is no longer selected and is removed from the `PlacementDecision`, the workload that is applied on this `ManagedCluster` is removed. You can prevent the workload removal by defining tolerations.

Reviewer: Similar to above suggestion, what do you think of more user focus here?: 
```suggestion
When you do not select `ManagedCluster` and you remove it from the `PlacementDecision`, the workload that is applied on this `ManagedCluster` is removed. You can prevent the workload removal by defining tolerations.
```

> See xref:../cluster_lifecycle/placement_filter.adoc#placement-tolerations[Filtering `ManagedClusters` by defining tolerations with placement] to learn more about defining tolerations.

Reviewer: For usability, I think it's easier to end the sentence with something for the users to click: 

```suggestion
To learn more about defining tolerations, see xref:../cluster_lifecycle/placement_filter.adoc#placement-tolerations[Filtering `ManagedClusters` by defining tolerations with placement].
```

[#placement-decision-resources]
== Additional resources

> -  See xref:../api/placementdecision.json.adoc#placementdecisions-api[PlacementDecisions API] to learn more about the API.

Reviewer: ```suggestion
-  To learn more about the API, see xref:../api/placementdecision.json.adoc#placementdecisions-api[PlacementDecisions API].
```
